OpenRA.Game.exe.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>

```

Comment:
If it is a problem for self-compiles, then we could have the windows dependencies script copy it to the root.